engine/common/components-tests/src/test/scala/pl/touk/nussknacker/engine/common/components/DecisionTableSpec.scala

```diff
@@ -173,6 +173,20 @@ trait DecisionTableSpec
                     )
                   )
                 )
+              ),
+              ExpressionParserCompilationError(
+                message = "There is no property 'age' in type: Record{}",
```

Comment:
Hmm this message for business people can be scary..  Why not:
`
There is no property 'age' in table?
`

Reply:
Agree. But the same message users will see in any component that has SpEL expression ;) I mean, it's not something that is specific in case of the Decision Table component. Probably can be improved, but it's rather related to SpEL validation improvements.

Reply:
or maybe ... I have one idea, how to provide a little improvement in this case inside the Decision Component. Will check

Reply:
@lciolecki check it out now ;)